iina/FileGroup.swift

```diff
@@ -27,7 +27,7 @@ class FileInfo: Hashable {
   var prefix: String {  // prefix detected by FileGroup
     didSet {
       if prefix.characters.count < self.characters.count {
-        suffix = filename.substring(from: filename.index(filename.startIndex, offsetBy: prefix.characters.count))
+        suffix = String(filename[filename.index(filename.startIndex, offsetBy: prefix.characters.count)...])
```

Comment:
You'd need to put a cast anyways, since `suffix(from:)` returns a `Substring` too:
```swift
suffix = String(filename.suffix(from: filename.index(filename.startIndex, offsetBy: prefix.characters.count)))
```
This makes it a bit longer than the slicing it with a subscript.